webapp/src/sagas.js

```diff
@@ -408,7 +410,7 @@ function getBidGroupsNonce(bidGroups) {
 function* handleFastBid(action) {
   const parcel = action.parcel
   if (parcel.projectId) {
-    // TODO: Show message, can't bid on project
+    yield call(() => flashNotice(`You can't bid on a project parcel`))
```

Comment:
This code is never reached as we are checking for:

![image](https://user-images.githubusercontent.com/692077/34041516-a06811dc-e198-11e7-8193-778ef0221bb1.png)

before bidding. That said I don't think it's a bad idea to have it here :)

Reply:
It's a side effect! flashNotice has to be here --that's what sagas are for